src/cli/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
-﻿# Changelog
+# Changelog
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 Prefix your items with `(Template)` if the change is about the template and not the resulting application.
 
+## 0.19.5
```

Comment:
It should be a minor update since this is a feature.

Pipeline is failing because `CHANGELOG.md doesn't mention current version (0.20.0).`.

Reply:
Yes.

Reply:
done